diagnostic.md

```diff
@@ -5,20 +5,24 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    nested routes
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    app/application/route.js  => load model to get data
+    app/application/template.hbs  => shows what to display on the home page with info on my-app
+    app/application/components/my-map/template.hbs  => what to display on the page when you are in the my-app url
+    app/application/components/my-app/component.js  => how you want to display the file and any actions you want to add
+    app/route.js  => maps your app to a url
```

Comment:
Looks good.  The `template.hbs` is also responsible for binding data, and binding and dispatching actions.